applications/directory/src/components/CreateProfileForm.js

```diff
@@ -516,12 +516,13 @@ const CreateProfileForm = ({
                 onBlur={async (e) => {
                   if (values.email) {
                     const isValidEmail = await ifEmailExists(values.email);
+      
                     // if email is valid set error message to empty string or populate with proper error message
-                    if (isValidEmail.validEmail) {
+                    if (!isValidEmail.profile) {
                       await setIfValidEmail('');
```

Comment:
Not sure why `await` is used here when setting a state variable.

Reply:
agree, do not even remember why I ended up doing so